src/components/Cards.jsx

```diff
@@ -9,10 +9,13 @@ export class Cards extends Component {
     return (
       <TouchableOpacity style={CardStyle.bottomCard} onPress={this.props.onPress}>
         <View style={CardStyle.topView}>
-          <Image
-            source={{ uri: this.props.img }}
-            style={CardStyle.imgCard}
-          />
+          {
+            this.props.img.length > 0 &&
```

Comment:
You can just use `this.props.img`instead of `this.props.img.length > 0` next time